hswindow/hstoolwindow/hsprojectexplorer/projectexplorer.go

```diff
@@ -138,13 +138,21 @@ func (m *ProjectExplorer) GetProjectTreeNodes() g.Layout {
 		return []g.Widget{g.Label("No project loaded...")}
 	}
 
-	fileStructure := m.project.GetFileStructure()
+	fileStructure, err := m.project.GetFileStructure()
+	if err != nil {
+		dialog.Message(err.Error()).Error()
+	}
```

Comment:
why, in cases like that, don't you `log`-`Print` errors?

Reply:
good point, changed.